client/src/components/modules/MapCard.js

```diff
@@ -25,11 +27,14 @@ class MapCard extends Component {
           <div className="MapCard-title">
             <div className={`MapCard-icon mod-${this.props.mod}`}></div>
             {`${this.props.mod}${this.props.index}`}
+            <div style={{ marginRight: 12 }} />
+            {this.props.customMap && (<img className="MapCard-overlay-icon" src={CustomMapBadge} />)}
+            {this.props.customSong && (<img className="MapCard-overlay-icon" src={CustomSongBadge} />)}
           </div>
         }
         bordered={true}
         cover={
-          <a target="_blank" href={`https://osu.ppy.sh/b/${this.props.mapId}`}>
+          <a target="_blank" href={`https://osu.ppy.sh/b/${this.props.mapId}`} style={{ position: "relative" }}>
```

Comment:
what is this `position: relative` for?

Reply:
Oops that's no longer needed as it was for the other design (overlay on bg), missed it